content/TruenasApps/StableApps/EmbyApp.md

```diff
@@ -0,0 +1,189 @@
+
+If you organize the **config** dataset under a parent dataset named **emby**, configure the [ACL permissions]({{< relref "PermissionsSCALE.md" >}}) for this parent dataset and add an ACE entry for the **root** user.
+
+You can add extra storage volumes during the app installation, or edit the application after it deploys. Stop the app before editing settings.
+{{< include file="/static/includes/apps/InstallAppStorageConfig2.md" >}}
+
+#### Setting Dataset ACL Permissions
+You can configure ACL permissions for the required dataset in the **Install Emby** wizard, or from the **Datasets** screen any time after adding the datasets.
+
+{{< include file="/static/includes/apps/InstallWizardStorageACLConfig.md" >}}
+
+{{< expand "Adding ACL Permissions from the Datasets Screen" "v">}}
+First, select the dataset row, scroll down to the **Permissions** widget, and click **Edit** to open the **Edit ACL** screen.
+Change the **@owner** and **@group** values from **root** to the administrative user for your TrueNAS system, and click apply for each.
+Next, add an ACL entry for the run-as user.
+For Emby, the run-as users are **0** for **root**. Add a user entry for this user.
+Save the ACL before leaving the screen.
+{{< /expand >}}
+
+#### Mounting an SMB Share Storage Volume
+TrueNAS **Additional Storage** options include the ability to mount an SMB share inside the container pod.
+{{< expand "Configuring Additional Storage Volumes" "v" >}}
+If you choose to configure additional storage volumes, click **Add** to the right of **Additional Storage** to show the **Type** field with three options:
+* **HostPath (Path that already exists on the system)** requires an existing dataset.
+* **ixVolume (Dataset created automatically by the system)** creates a storage volume in the hidden **ix-apps** dataset.
+* **SMB/CIFS Share (Mounts a volume to a SMB share)** shows settings to create an SMB share storage volume.
+  You must create the SMB share user and share dataset before adding this type.
+* **Anonymous (Temporary directory created on the disk)** creates a temporary directory in the hidden **ix-apps** dataset.
+* **Tmpfs (Temporary directory created on the RAM)** creates a temporary directory in RAM.
+{{< /expand >}}
+
+If adding an SMB share as an additional storage volume, create the SMB dataset and share user(s), and add the user ID for the share user(s) to the dataset ACL.
+{{< include file="/static/includes/apps/InstallWizardStorageSMBOption.md" >}}
+
+### Labels Configuration
+Emby uses metadata to add information to media files, such as where media files are saved, the language used, and parental controls.
+Emby uses tags to add identification information to media files.
+
+{{< include file="/static/includes/apps/InstallWizardLabelsConfiguration.md" >}}
+
+{{< trueimage src="/images/SCALE/Apps/InstallEmbyLabelsConfig.png" alt="Labels Configuration Settings" id="Labels Configuration Settings" >}}
```

Comment:
You use the same image for labels as you do for resources

Reply:
fixed